core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,19 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive( 'updateLanguage', updateLanguage );
```

Comment:
Hi @SudhanvaMG -- just a general comment, please could you rewrite this to follow the style of the surrounding code? It's important, in a shared codebase, for the overall style to be consistent. I'm seeing a bunch of extra spaces all over the place, the indentation level is wrong, and this directive is not declared in the same way as the other existing directives (which don't declare a second updateLanguage function). It isn't enough to get something that seems to work correctly; style matters too, since other people may be maintaining your code.

Thanks!